lua/conform/formatters/vsg.lua

```diff
@@ -0,0 +1,38 @@
+local function find_local_config()
+  local current_file = vim.api.nvim_buf_get_name(0)
+  local local_configs = { ".vsg.yaml", ".vsg.yml", ".vsg.json" }
+  return vim.fs.find(local_configs, {
+    path = vim.fs.dirname(current_file),
```

Comment:
Instead of using the current buffer, the args function should take the parameters `(self, ctx)` and you should use `ctx.dirname` for this